docs/api/IoT.js-API-Assert.md

```diff
@@ -5,203 +5,234 @@ The following shows Assert module APIs available for each platform.
 |  | Linux<br/>(Ubuntu) | Raspbian<br/>(Raspberry Pi) | Nuttx<br/>(STM32F4-Discovery) |
 | :---: | :---: | :---: | :---: |
 | assert.assert | O | O | O |
-| assert.doesNotThrows | O | O | O |
+| assert.doesNotThrow | O | O | O |
 | assert.equal | O | O | O |
 | assert.fail | O | O | O |
 | assert.notEqual | O | O | O |
 | assert.notStrictEqual | O | O | O |
 | assert.strictEqual | O | O | O |
 | assert.throws | O | O | O |
 
+# Assert
 
-### Contents
+Assert module is for writing tests for applications.
 
-- [Assert](#assert)
-    - [Module Functions](#module-functions)
-        - [`assert.assert(value[, message])`](#assertassertvalue-message)
-        - [`assert.doesNotThrows(block[, message])`](#assertdoesnotthrowsblock-message)
-        - [`assert.equal(actual, expected[, message])`](#assertequalactual-expected-message)
-        - [`assert.fail(actual, expected, message, operator)`](#assertfailactual-expected-message-operator)
-        - [`assert.notEqual(actual, expected[, message])`](#assertnotequalactual-expected-message)
-        - [`assert.notStrictEqual(actual, expected[, message])`](#assertnotstrictequalactual-expected-message)
-        - [`assert.strictEqual(actual, expected[, message])`](#assertstrictequalactual-expected-message)
-        - [`assert.throws(block[, expected, message])`](#assertthrowsblock-expected-message)
-    - [Class: assert.AssertionError](#class-assertassertionerror)
-        - [Properties](#properties)
-            - [`actual`](#actual)
-            - [`message`](#message)
-            - [`expected`](#expected)
-            - [`name`](#name)
-            - [`operator`](#operator)
+You can access the functions of the module by adding `require('assert')` to your file.
 
 
-# Assert
+## Class: AssertionError
+
+Assert module will produce `AssertionError` in case of an assertion failure. `AssertionError` inherits standard `Error` thus it has properties provided by `Error` object including additional properties.
+
+
+### Properties
+
+
+* `actual` {any} This property contains the actual value.
 
-Assert module is for writing test for applications.
+* `expected` {any} This property contains the expected value.
 
-You can get assert module by `require('assert')` the module provide.
+* `message` {any} The error message, default value is the error itself.
+
+* `name` {String} The name is set AssertionError.
+
+* `operator` {String} This property contains the operator used for evaluating `actual` with `expected`.
 
 
 ## Module Functions
 
 
-### `assert.assert(value[, message])`
-* `value <any>`
-* `message <any>`
+### assert(value[, message])
+* `value` {any} Value to test.
+* `message` {any} Message displayed in the thrown error.
 
-Test if `value` is true, it throws an `AssertionError` exception if `value` evaluates `false`.
+Checks if the `value` is truthy. If not, throws an AssertionError, with the given `message`.
```

Comment:
I don't think there is a problem with truthy.
In JavaScript, a truthy value is a value that is considered  true when evaluated in a Boolean context.
> If it does not

`If it is not` would be the right phrase, as you are referring to the value, not the check itself.